sys/posix/sockets/posix_sockets.c

```diff
@@ -795,7 +795,7 @@ ssize_t recvfrom(int socket, void *restrict buffer, size_t length, int flags,
         case SOCK_RAW:
             /* TODO: apply configured timeout */
             if ((res = sock_ip_recv(&s->sock->raw, buffer, length, SOCK_NO_TIMEOUT,
```

Comment:
Personally, I don't see any benefit, but feel free to provide a PR against my PR, but then please update all following cases accordingly for consistency.